### PR TITLE
get author name and category label

### PR DESCRIPTION
**Pull request by devjoaov**
missing:

- dockerfile with python and poetry
- add workers and server to docker-compose.yml
- add recurrent async tasks to celery-beat schedule
- map UI categories with DB categories
- filter by category
- fix/udpate `forum/post` page

### CODE REVIEW

File: data/discourse/categories.json [hunk 1]
```diff
@@ -0,0 +1,744 @@
+{
```
Comment (ribeirojose) [inline data/discourse/categories.json:1]: ❌ let's just add a script that adds this to the db

File: pkg/op-forum-agg/op_forum_agg/worker.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+def setup_periodic_tasks(sender, **kwargs):
+
+    # Executes every Monday morning at 7:30 a.m.
+    # sender.add_periodic_task(
+    #     crontab(hour=7, minute=30, day_of_week=1),
+    #     long_task,
+    # )
+
+    # each 10 seconds
+    sender.add_periodic_task(10, long_task.s(), name="sync categories")
```
Comment (ribeirojose) [inline pkg/op-forum-agg/op_forum_agg/worker.py:44]: tbh each day would be more than enough here

File: pkg/op-forum-agg/op_forum_agg/src/utils.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        base_node = self._extractor.node_from_url(url)
+
+        if isinstance(base_node, Board):
+            self.write_board(base_node)
+        elif isinstance(base_node, Thread):
+            self.write_thread(base_node)
+
+        return self.get_entries()
+
+    def read_metadata(self):
```
Comment (ribeirojose) [inline pkg/op-forum-agg/op_forum_agg/src/utils.py:56]: what would this do?

File: pkg/op-forum-agg/op_forum_agg/src/sync/threads.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    forum_categories = site_info_raw_json["categories"]
+
+    for category in forum_categories[1:]:
+        try:
+            print(
+                "Starting to fetch threads for",
+                category["slug"],
+                datetime.now(timezone.utc),
+            )
+            # this should be an async task
```
Comment (ribeirojose) [inline pkg/op-forum-agg/op_forum_agg/src/sync/threads.py:92]: 👍

**Comment by devjoaov** [comment]
> > missing:
> > 
> > * dockerfile with python and poetry
> > * add workers and server to docker-compose.yml
> > * add recurrent async tasks to celery-beat schedule
> > * map UI categories with DB categories
> > * filter by category
> > * fix/udpate `forum/post` page
> 
> can you ensure we create task for these?

https://linear.app/bleu-llc/issue/OP-50/govsummarizer-ui

**Pull request merged**